File: models/Ritual.js
const mongoose = require('mongoose');

const RitualSchema = new mongoose.Schema({
    title: {
        type: String,
        required: true
    },
    title_ta: {
        type: String, // Tamil title
        required: false
    },
    subtitle: {
        type: String,
        required: true
    },
    subtitle_ta: {
        type: String, // Tamil subtitle
        required: false
    },
    description: {
        type: String,
        required: false
    },
    description_ta: {
        type: String,
        required: false
    },
    imageUrl: {
        type: String,
        required: true
    },
    price: {
        type: Number,
        default: 0
    },
    isActive: {
        type: Boolean,
        default: true
    },
    order: {
        type: Number,
        default: 0
    }
}, { timestamps: true });

module.exports = mongoose.model('Ritual', RitualSchema);

File: astroeleven/android/app/src/main/java/com/astroeleven/app/data/model/Ritual.kt
package com.astroeleven.app.data.model

import com.google.gson.annotations.SerializedName

data class Ritual(
    @SerializedName("_id") val id: String,
    @SerializedName("title") val title: String,
    @SerializedName("title_ta") val titleTamil: String? = null,
    @SerializedName("subtitle") val subtitle: String,
    @SerializedName("subtitle_ta") val subtitleTamil: String? = null,
    @SerializedName("description") val description: String? = null,
    @SerializedName("description_ta") val descriptionTamil: String? = null,
    @SerializedName("imageUrl") val imageUrl: String,
    @SerializedName("price") val price: Double = 0.0,
    @SerializedName("isActive") val isActive: Boolean = true,
    @SerializedName("order") val order: Int = 0
)

data class RitualResponse(
    val ok: Boolean,
    val data: List<Ritual>
)
